setup.cfg

```diff
@@ -29,6 +29,7 @@ xfail_strict = true
 filterwarnings = 
 	ignore::UserWarning
 	ignore::FutureWarning
+	ignore:distutils Version classes are deprecated:
```

Comment:
What code is bringing about this warning?

Reply:
both xarray and setuptools:

<details>

```
================================================================= warnings summary =================================================================
../../../../miniconda3/envs/aics/lib/python3.9/site-packages/xarray/core/pycompat.py:22
aicsimageio/tests/readers/extra_readers/test_bioformats_reader.py::test_bioformats_reader[DICOM_samples_MR-MONO2-8-16x-heart.dcm-PRIMARY-expected_scenes16-expected_shape16-uint8-TCZYX-expected_channel_names16-expected_physical_pixel_sizes16-LOCAL]
aicsimageio/tests/readers/extra_readers/test_bioformats_reader.py::test_bioformats_reader[DICOM_samples_MR-MONO2-8-16x-heart.dcm-PRIMARY-expected_scenes16-expected_shape16-uint8-TCZYX-expected_channel_names16-expected_physical_pixel_sizes16-LOCAL]
aicsimageio/tests/readers/extra_readers/test_bioformats_reader.py::test_bioformats_reader[DICOM_samples_MR-MONO2-8-16x-heart.dcm-PRIMARY-expected_scenes16-expected_shape16-uint8-TCZYX-expected_channel_names16-expected_physical_pixel_sizes16-LOCAL]
  /Users/talley/miniconda3/envs/aics/lib/python3.9/site-packages/xarray/core/pycompat.py:22: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    duck_array_version = LooseVersion(duck_array_module.__version__)

../../../../miniconda3/envs/aics/lib/python3.9/site-packages/xarray/core/pycompat.py:37
../../../../miniconda3/envs/aics/lib/python3.9/site-packages/xarray/core/pycompat.py:37
  /Users/talley/miniconda3/envs/aics/lib/python3.9/site-packages/xarray/core/pycompat.py:37: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    duck_array_version = LooseVersion("0.0.0")

../../../../miniconda3/envs/aics/lib/python3.9/site-packages/setuptools/_distutils/version.py:351
../../../../miniconda3/envs/aics/lib/python3.9/site-packages/setuptools/_distutils/version.py:351
../../../../miniconda3/envs/aics/lib/python3.9/site-packages/setuptools/_distutils/version.py:351
../../../../miniconda3/envs/aics/lib/python3.9/site-packages/setuptools/_distutils/version.py:351
  /Users/talley/miniconda3/envs/aics/lib/python3.9/site-packages/setuptools/_distutils/version.py:351: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    other = LooseVersion(other)

../../../../miniconda3/envs/aics/lib/python3.9/site-packages/xarray/core/npcompat.py:82
  /Users/talley/miniconda3/envs/aics/lib/python3.9/site-packages/xarray/core/npcompat.py:82: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    if LooseVersion(np.__version__) >= "1.20.0":

../../../../miniconda3/envs/aics/lib/python3.9/site-packages/xarray/core/pdcompat.py:45
  /Users/talley/miniconda3/envs/aics/lib/python3.9/site-packages/xarray/core/pdcompat.py:45: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    if LooseVersion(pd.__version__) < "0.25.0":

-- Docs: https://docs.pytest.org/en/stable/warnings.html
```

</details>

Reply:
Ahh gotcha. I've been getting a lot of those `LooseVersion` warnings on other projects. Good to know I can borrow this ignoration 😁